coupling/delwaq/test.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from generate import generate
+from parse import parse
+from util import run_delwaq
+
+delwaq_dir = Path(__file__).parent
+
+
+def test_offline_delwaq_coupling():
+    repo_dir = delwaq_dir.parents[1]
+    modelfn = repo_dir / "generated_testmodels/basic/ribasim.toml"
+
+    graph, substances = generate(modelfn)
+    run_delwaq()
+    model = parse(modelfn, graph, substances)
+    df = model.basin.concentrationexternal
```

Comment:
Shouldn't this be `concentration_external`, like here?

https://github.com/Deltares/Ribasim/pull/1267/files#diff-bdece7de80ef751956236e71b5aed3964311d59479f076c892d6745b0e30266dR298